packages/flutter/test/material/theme_data_test.dart

```diff
@@ -216,7 +216,6 @@ void main() {
     );
 
     final ThemeData theme = ThemeData.raw(
```

Comment:
Do we need a test to check that the brightness is set correctly with the changes to the constructor?

Reply:
Good point; I will add that.